lib/assessment.js

```diff
@@ -8,33 +8,87 @@
 // Question 1
 // Write a callback for reduce that selects
 // the minimum value in the array
-var min = function min() {
+var min = function min() {previousValue, currentValue, index, array)
```

Comment:
Your `min` function is right-on except for a bit of syntax. This line I have just called out should be:

`var min = function min(previousValue, currentValue, index, array) {`

Note the brackets. Previously, you had an empty set of parens followed by a mismatched curly-paren pair, and no open curly bracket at the end to match the closing curly bracket after the function body.

Here is the template for function syntax:

```js
function name(param1, param2, ..., paramN) {
    statement1;
    statement2;
    ...
    statementN;
}
```

9/10 credit for being correct except for syntax.